src/help.sh

```diff
@@ -70,7 +70,18 @@ function kworkflow_man()
 
 function kworkflow_version()
 {
-  local version_path="$KW_LIB_DIR/VERSION"
+  local head_hash
+  local branch_name
+  local base_version
+  local version_path="${KW_LIB_DIR}/VERSION"
 
-  cat "$version_path"
+  if [[ "${KW_REPO_MODE}" == 'y' ]]; then
```

Comment:
Just as a curiosity, when you are only expanding a variable (not concatenating it with other literals/expansions) you do not need to use the curly braces, i.e., you can use `"$KW_REPO_MODE"`.

Reply:
Yes, but I like curly braces, haha.